docs/products/kafka/kafka-connect/howto/debezium-source-connector-sql-server.rst

```diff
@@ -0,0 +1,193 @@
+* ``APACHE_KAFKA_PORT``: The port of the Apache Kafka service, needed when storing the :ref:`schema definition changes <connect_debezium_sql_server_schema_versioning>`
+* ``SCHEMA_REGISTRY_PORT``: The Apache Kafka's schema registry port, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_USER``: The Apache Kafka's schema registry username, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_PASSWORD``: The Apache Kafka's schema registry user password, only needed when using Avro as data format
+
+
+.. Note::
+
+    If you're using Aiven for SQL Server and Aiven for Apache Kafka the above details are available in the `Aiven console <https://console.aiven.io/>`_ service Overview tab or via the dedicated ``avn service get`` command with the :ref:`Aiven CLI <avn_service_get>`.
+
+Setup a SQL Server Debezium source connector with Aiven Console
+---------------------------------------------------------------
+
+The following example demonstrates how to setup a Debezium source Connector for Apache Kafka to a SQL Server database using the `Aiven Console <https://console.aiven.io/>`_.
+
+Define a Kafka Connect configuration file
+'''''''''''''''''''''''''''''''''''''''''
+
+Define the connector configurations in a file (we'll refer to it with the name ``debezium_source_sql_server.json``) with the following content, creating a file is not strictly necessary but allows to have all the information in one place before copy/pasting them in the `Aiven Console <https://console.aiven.io/>`_:
+
+.. code-block:: json
+
+    {
+        "name":"CONNECTOR_NAME",
+        "connector.class": "io.debezium.connector.sqlserver.SqlServerConnector",
+        "database.hostname": "SQLSERVER_HOST",
+        "database.port": "SQLSERVER_PORT",
+        "database.user": "SQLSERVER_USER",
+        "database.password": "SQLSERVER_PASSWORD",
+        "database.dbname": "SQLSERVER_DATABASE_NAME",
+        "database.server.name": "KAFKA_TOPIC_PREFIX",
+        "table.include.list": "SQLSERVER_TABLES",
+        "tasks.max":"NR_TASKS",
+        "poll.interval.ms": 500,
+        "key.converter": "io.confluent.connect.avro.AvroConverter",
+        "key.converter.schema.registry.url": "https://APACHE_KAFKA_HOST:SCHEMA_REGISTRY_PORT",
+        "key.converter.basic.auth.credentials.source": "USER_INFO",
+        "key.converter.schema.registry.basic.auth.user.info": "SCHEMA_REGISTRY_USER:SCHEMA_REGISTRY_PASSWORD",
+        "value.converter": "io.confluent.connect.avro.AvroConverter",
+        "value.converter.schema.registry.url": "https://APACHE_KAFKA_HOST:SCHEMA_REGISTRY_PORT",
+        "value.converter.basic.auth.credentials.source": "USER_INFO",
+        "value.converter.schema.registry.basic.auth.user.info": "SCHEMA_REGISTRY_USER:SCHEMA_REGISTRY_PASSWORD",
+        "database.history.kafka.bootstrap.servers": "APACHE_KAFKA_HOST:APACHE_KAFKA_PORT",
```

Comment:
Here we also need   "database.history.kafka.topic": "history", as in mysql example